src/AppInstallerCLICore/Argument.cpp

```diff
@@ -183,6 +183,14 @@ namespace AppInstaller::CLI
             return { type, "enable"_liv, ArgTypeCategory::None, ArgTypeExclusiveSet::StubType };
         case Execution::Args::Type::ConfigurationDisable:
             return { type, "disable"_liv, ArgTypeCategory::None, ArgTypeExclusiveSet::StubType };
+        case Execution::Args::Type::ConfigurationAllUsersLocation:

```

Comment:
What if we had one argument and it had special case text rather than all 4?
```
--module-path currentuser
--module-path allusers
--module-path default
--module-path Z:\my_dir
```
If you really want to use a directory named `allusers`, you just fully qualify it `D:\allusers`.

Alternately, I'm not really sure why we need to expose it at all at the moment.